src/targets/ref/lowering.cpp

```diff
@@ -885,9 +886,41 @@ struct ref_rnn_var_sl_last_output
 };
 MIGRAPHX_REGISTER_OP(ref_rnn_var_sl_last_output)
 
+struct ref_if
```

Comment:
Is this necessary? Why cant the ref use the original `if` op's `compute` method?

Reply:
changed accordingly